runtime/src/accounts_background_service.rs

```diff
@@ -441,9 +441,19 @@ mod test {
         bank0.store_account(&account_key, &Account::new(264, 0, &Pubkey::default()));
         assert!(bank0.get_account(&account_key).is_some());
         pruned_banks_sender.send(0).unwrap();
+
+        assert!(!bank0
+            .rc
+            .accounts
+            .scan_slot(0, |stored_account| Some(stored_account.stored_size()))
```

Comment:
nits: maybe replace `Some(stored_account.stored_size())` with `unreachable!()` to better convey this closure doesn't matter?

Reply:
yeah the contents of the closure don't matter, but the closure will still be executed. Replaced with `()`